spec/std/random/system_spec.cr

```diff
@@ -0,0 +1,10 @@
+require "spec"
+require "random/system"
+
+describe "Random::System" do
+  rng = Random::System.new
+
+  it "returns random numbers from the secure system source" do
+    typeof(rng.next_u).should eq(UInt32)
```

Comment:
The implementation as presented in this PR happens to have `next_u` returning an `UInt32` but I don't think it should be part of the *specification*. `module Random` does not require `next_u` to return an `UInt32`, anyway.